main.c

```diff
@@ -112,19 +113,40 @@ void print_rolls(int *dice_nums) {
     if(!print_separate) printf("\n");
 }
 
+/* roll_from_interactive() - parse stdin, one roll by line, don't exit on errors
+ *
+ * Parameters: None
+ * Returns: None
+ */
+void roll_from_interactive(){
+     int dice_nums[DICE_ARRAY_SIZE] = {};
+     static char *line = (char *)NULL;
+
+     line = readline("");
+     while(line){
+       parse_string( line, dice_nums );
+
+       free(line);
+       print_rolls(dice_nums);
+
+       line = (char *)NULL;
+       line = readline("");
```

Comment:
Same here. Set it up as a #define or something, so the magic constant string won't appear multiple times and thus we'll remember later which calls to readline get the interactive prompt vs. those we really do want to be promptless (stdin mode).